examples/bbengfort/testing.ipynb

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
To continue the demo of how pull requests from new users might go:

Looks like a good start at a notebook!